public/js/listeners.js

```diff
@@ -272,8 +274,9 @@ function startListeners() {
         } else {
             data = e.clipboardData.getData('text/plain');
         }
-
-        paste(data);
+        // Check if data occurs on local storage else take data from clipboard
+        if (!(localStorage.getItem('clipboardData'))) localStorage.setItem('clipboardData', data);
+        paste(localStorage.getItem('clipboardData'));
```

Comment:
I don't think is a good idea. Once copied, it will persist on localstorage forever so basically clipboard is never used again.